pyproject.toml

```diff
@@ -49,6 +49,7 @@ show_error_codes = true
 exclude = ["docs", "test"]
 strict = true
 namespace_packages = true
+ignore_missing_imports = true
```

Comment:
Is this required because we split `app.py`?

Reply:
This is used for many import, I am not exactly why mypy is complaining, but it will complains there is [missing import](https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports).

This happens for imports such as:

```
from app_utils.monitoring import post_response_to_metric_service
from app_utils.chroma import connect_vector_store, query_vector_store
from app_utils.llm import build_memory_dict, query_llm
from app_utils.endpoints import get_metric_service_endpoint, get_prediction_endpoint
```

Reply:
One thing to try would be using `from app.app_utils ..` instead of `from app_utils`. Let me know if this works.

Reply:
Can you also replace using above import i.e. append `app.` before all imports?

Reply:
That didn't work, streamlit failed to start on k8s.

<img width="788" alt="Screenshot 2023-08-31 at 10 17 24" src="https://github.com/fuzzylabs/MindGPT/assets/25309418/aed6c383-c251-4079-869b-8594c900b210">